apps/opencs/model/world/refidadapterimp.cpp

```diff
@@ -312,7 +312,7 @@ QVariant CSMWorld::BookRefIdAdapter::getData (const RefIdColumn *column,
     const Record<ESM::Book>& record = static_cast<const Record<ESM::Book>&> (
         data.getRecord (RefIdData::LocalIndex (index, UniversalId::Type_Book)));
 
-    if (column==mScroll)
+    if (column==mBookType)
         return record.get().mData.mIsScroll!=0;
```

Comment:
Yes, I wasn't sure about what do change now and what to leave for later. As of now, everything relies on boolean behaviour. I'll adjust the code.